API/Controllers/CustomerController.cs

```diff
@@ -62,11 +61,11 @@ public IActionResult Create(Customer customer)
         }
 
         [HttpDelete]
```

Comment:
O Delete está sendo feito através de um Id, portanto é recomendado que o mesmo seja passado como parâmetro, similar ao GetById.
```suggestion

        [HttpDelete("{id}")]
```